lib/ace/selection.js

```diff
@@ -65,19 +65,19 @@ var Selection = function(session) {
     this.doc = session.getDocument();
 
     this.clearSelection();
-    this.lead = this.selectionLead = this.doc.createAnchor(0, 0);
-    this.anchor = this.selectionAnchor = this.doc.createAnchor(0, 0);
```

Comment:
selectionLead, selectionAnchor were deprecated for a while, and having `cursor ` instead of lead makes more sense since we have getCursor method